fairlearn/metrics/_group_metric_set.py

```diff
@@ -64,6 +70,27 @@
 SPECIFICITY_SCORE = "specificity_score"
 ZERO_ONE_LOSS = "zero_one_loss"
 
+
+class _func_wrapper:
+    """
+    Metric function wrapper, so they return 0 rather than raising ValueError.
```

Comment:
Does 0 make sense? I'm thinking this could be misleading... if my dataset doesn't have 0s perhaps it's just not defined (NaN? "Undefined"?) rather than 0. Wdyt?

Reply:
Can the dashboard cope with `np.NaN`? If so, I'd be happy with that.

Reply:
I've just checked @romanlutz , and if I substitute `np.nan` then the upload goes OK, but the dashboard itself doesn't load.

Reply:
That sounds like a shortcoming of the dashboard that needs fixing. 0 still doesn't make sense, does it?

Reply:
I agree that it's not great, but zero does have the benefit of being a valid `roc_auc_score()` and a _bad_ `roc_auc_score()`. It is going to give a 'balanced accuracy score difference' of zero, which is unfortunate; I'm not sure what will happen with the ratio when it hits the TS code and a divide-by-zero is thrown.

Reply:
This is also copying the default behaviour of `recall_score()` which can suffer a similar problem. Unlike `roc_auc_score()` though, `recall_score()` does provide a `zero_division` argument to allow users to control what happens.

Reply:
Ping.... @romanlutz ? I think that '0' is the best default for now - it's what `recall_score()` will also be returning in this sort of situation (and _has_ been returning to date).